src/main/java/org/tron/core/config/args/Args.java

```diff
@@ -330,6 +330,10 @@
   @Setter
   private String logLevel;
 
+  @Getter
+  @Setter
+  private int blockVersion = 3;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Assign this magic number 3 to a well-named constant, and use the constant instead. ([squid:S109](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS109))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1664&issues=AWbKNvRYO08i52vdhiRp&open=AWbKNvRYO08i52vdhiRp)

[ISSUE_KEY]: AWbKNvRYO08i52vdhiRp